File: onboardUser/scripts/python/gen_account.py
from eth_account import Account

account = Account.create()

print("Account address: ", account.address)

# Clear the .env file and write the new signing key to the .env file
with open('.env', 'w') as f:
    f.write(f"export SIGNING_KEY='{account._private_key.hex()}'\n")

File: onboardUser/scripts/js/gen_account.mjs
import fs from 'fs';
import {privateToAddress} from 'ethereumjs-util';
import {generateECDSAPrivateKey} from 'soda-sdk';

// Generate ECDSA private key
const privateKey = generateECDSAPrivateKey().toString('hex');

// Create account address from private key
const address = privateToAddress(Buffer.from(privateKey, 'hex')).toString('hex');

console.log("Account address: ", '0x'+ address);

// Write the data to a .env file
fs.appendFileSync('.env', `export SIGNING_KEY='0x${privateKey}'\n`);
